suites/reef/rados/tier-2_rados_test_parameters.yaml

```diff
@@ -0,0 +1,90 @@
+#################################################################################
```

Comment:
Any reason for a New test suite? Can't we add these tests to existing suite?

Reply:
This suite I added in Quincy and not included in the reef. So I added this in Reef. The test cases are all related to checking parameters and execution will take around 5 hours.  That is the reason I kept it separate.